include/up-cpp/datamodel/constants/UuidConstants.h

```diff
@@ -25,8 +25,8 @@ constexpr uint64_t UUID_COUNTER_MASK = 0xFFF;
 constexpr uint64_t UUID_RANDOM_MASK = 0x3FFFFFFFFFFFFFFF;
 
 // Constants for UUID version and variant
-constexpr uint8_t UUID_VERSION_8 = 8;
-constexpr uint8_t UUID_VARIANT_RFC4122 = 2;
+constexpr uint64_t UUID_VERSION_7 = 7;
+constexpr uint64_t UUID_VARIANT_RFC4122 = 2;
```

Comment:
Is this variant correct? At least in [RFC9562](https://www.rfc-editor.org/rfc/rfc9562#section-4.1), it looks like it should actually be in the range `[0x8, 0xB]`